### PR TITLE
Gpu-card-component

### PR DESCRIPTION
**Pull request by dnnysng**
Removed default template stuff, added a css reset, added GpuListings component, and built GpuCard component with dummy data.

### CODE REVIEW

File: client/components/gpu-listings/GpuCard.module.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+.divider {
+  color: var(--off-white);
+  opacity: 30%;
+  margin: 1rem 0;
+}
+
+.brand {
+  text-transform: uppercase;
+  font-size: 01rem;
```
Comment (RealMattOwen) [inline client/components/gpu-listings/GpuCard.module.css]: Should this be 0.1rem or just 1rem?

File: client/components/gpu-listings/GpuCard.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        className={style.image}
+        height={200}
+        width={200}
+        alt="gpu thumbnail"
+      />
+      <hr className={style.divider} />
+      <div>
+        <span className={style.brand}>ASUS</span>
+        <h3 className={style.title}>ASUS ROG Strix NVIDIA GeForce RTX 3080 Ti OC Edition Gaming Graphics Card</h3>
+        <dl className={style.specs}>
```
Comment (RealMattOwen) [inline client/components/gpu-listings/GpuCard.js]: I can see the thinking behing using a `dl` here but semantically this should be a `ul` as you're using `li`'s

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl

The note regarding "Permitted content"

File: client/components/gpu-listings/GpuCard.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import style from './GpuCard.module.css'
+import Image from 'next/image'
```
Comment (RealMattOwen) [inline client/components/gpu-listings/GpuCard.js:1]: It's typically a common practice to sort imports in an order that is visually easier to scan, for example
```
<npm packages>

<your component imports>
<your icon / img imports>
<your util function imports>

<your style imports>
```

**Review by RealMattOwen — CHANGES_REQUESTED** [review]
Good job, especially on your first PR 💪

File: client/components/gpu-listings/GpuCard.js (hunk 3 above)
Comment (dnnysng) [reply client/components/gpu-listings/GpuCard.js:1]: Ok sounds good, I will update and do that moving forward.

File: client/components/gpu-listings/GpuCard.js (hunk 2 above)
Comment (dnnysng) [reply client/components/gpu-listings/GpuCard.js]: That makes sense, I will fix this.

File: client/components/gpu-listings/GpuCard.module.css (hunk 1 above)
Comment (dnnysng) [reply client/components/gpu-listings/GpuCard.module.css]: This should be 1, I must of missed the 0 lol

**Review by RealMattOwen — APPROVED** [review]

**Pull request merged**